### DISCUSSION TITLE
New Trigger: SpriteNo

### DISCUSSION THREAD

**Discussion by LasombraDemon** [ikemen-engine/Ikemen-GO#1288] [Feature Request]
### Is your feature request related to a problem? Please describe.

Now we can refer only to StateNos and AnimNos, but a SpriteNo trigger would be useful, too.

When using customstates and their own animations, other characters could put our own in certain sprites, and we cannot keep track of that. This could be useful.

### Describe the solution you'd like

SpriteNo could return Sprite.Index as a decimal separated Float; this could solve the whole "Group,Index" issue, where it should actually be treated as a String.

### Search terms

Trigger SpriteNo

**Comment by potsmugen** [comment]
Implemented in AnimFrame trigger.
